Language/bg/Images.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+// Images language settings
+return [
+    'sourceImageRequired'    => 'Трябва да посочите изходно изображение във вашите предпочитания.',
+    'gdRequired'             => 'Необходима е GD библиотека с изображения, за да използвате тази функция.',
+    'gdRequiredForProps'     => 'Вашият сървър трябва да поддържа GD библиотека с изображения, за да определи свойствата на изображението.',
+    'gifNotSupported'        => 'GIF изображенията често не се поддържат поради лицензионни ограничения. Може да се наложи да използвате JPG или PNG изображения вместо това.',
+    'jpgNotSupported'        => 'JPG изображения не се поддържат.',
+    'pngNotSupported'        => 'PNG изображенията не се поддържат.',
+    'webpNotSupported'       => 'WEBP изображенията не се поддържат.',
+    'fileNotSupported'       => 'Доставеният файл не е поддържан тип изображение.',
+    'unsupportedImageCreate' => 'Вашият сървър не поддържа GD функцията, необходима за обработка на този тип изображение.',
+    'jpgOrPngRequired'       => 'Протоколът за преоразмеряване на изображението, посочен във вашите предпочитания, работи само с типове изображения JPEG или PNG.',
+    'rotateUnsupported'      => 'Изглежда ротацията на изображението не се поддържа от вашия сървър.',
+    'libPathInvalid'         => 'Пътят до вашата библиотека с изображения не е правилен. Моля, задайте правилния път в предпочитанията си за изображение. {0}',
+    'imageProcessFailed'     => 'Неуспешна обработка на изображението. Моля, проверете дали вашият сървър поддържа избрания протокол и че пътят до вашата библиотека с изображения е правилен.',
+    'rotationAngleRequired'  => 'Необходим е ъгъл на завъртане за завъртане на изображението.',
+    'invalidPath'            => 'Пътят до изображението не е правилен.',
+    'copyFailed'             => 'Програмата за копиране на изображението е неуспешна.',
+    'missingFont'            => 'Не може да се намери шрифт за използване.',
+    'saveFailed'             => 'Не може да се запази изображението. Моля, уверете се, че изображението и файловата директория могат да се записват.',
+    'invalidDirection'       => 'Посоката на обръщане може да бъде само `вертикална` или `хоризонтална`. Дадено: {0}',
+    'exifNotSupported'       => "Четенето на EXIF \u{200b}\u{200b}данни не се поддържа от тази PHP инсталация.",
```

Comment:
Please remove the zero width space, then re-run `vendor/bin/php-cs-fixer fix -v`

Reply:
Good catch, fixed.